assets/app/view/game/corporation.rb

```diff
@@ -66,7 +66,7 @@ def render
 
         children = [render_title, render_holdings]
 
-        unless @corporation.minor?
+        unless @game.corporation_is_minor?(@corporation)
```

Comment:
why?

Reply:
Does it break a lot of games if you add `corp.type == :minor` to the `corporation.minor?` logic?

Reply:
This game uses the 1822 style of minors. This means that corporation? is true, minor? is false, but type is set to :minor

I want these to display like minors, but I didn't want to affect how 1822 worked, so I created and used a game method corporation_is_minor? to adjust the appearance in the UI.

Reply:
There are a ton of games that use corp.type == :minor, not just 1822. I worry this method will increase the confusion between the two styles of minors. How about creating a method like show_full_coporation_card?(corporation) or something like that to determine if the share price, ownership, etc. is shown on the card. I think that's more specific to what you are trying to do.

Reply:
that doesn’t make sense, why isn’t it just a minor then?

Reply:
@tobymao - I can't speak to the other games, but in 18NewEngland, I need a minor that has a share price and could be parred instead of merely bought. Please bear in mind, I didn't create this type of minor. It already existed and I'm just leveraging it.

@crericha - Is your main concern about the method name? I can change it.

Reply:
1822-style minors that have a share price and various train buying/owning features are the norm, not 46-style private company minors.

Reply:
I followed @crericha's suggestion and renamed the corporation_is_minor? method

Reply:
Looks good to me.

The other thing I asked is if you tried to modify the minor? method by adding the type == :minor logic to it.

Reply:
I thought about it, but that would alias the two minor variations together. I didn't want to break "type" minors in other games.

Reply:
This may be a good idea, but I don't want to bog down this game with refactoring a bunch of other games.

Reply:
To answer your question specifically, that change would affect how minors in 1822 are displayed.